tests/v2/validation/provisioning/rke2/config.go

```diff
@@ -13,6 +14,7 @@ const (
 
 type Config struct {
 	NodesAndRoles      []machinepools.NodeRoles `json:"nodesAndRoles" yaml:"nodesAndRoles" default:"[]"`
+	NodesAndRolesRKE1  []nodepools.NodeRoles    `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1" default:"[]"`
```

Comment:
Is it necessary to have two versions of this Markus? And also this config file would fit better outside of rke2 and just in v2/validation/provisioning since both rke1 and rke2 both use it.

Reply:
Nevermind I like that they are different, I think that works fine. But config.go should definitely go in the parent directory and not in rke2.

Reply:
Since we're moving config.go to the parent directory, does it make sense to do that for providers.go as well? Or do we think it's best to keep that inside of the `rke2` folder?

Reply:
Yes move that as well. Anything that both rke2/rke1 use.

Reply:
Never mind, neglected to remember that both `providers.go` are vastly different. Pushed the changes and running freeform jobs for RKE1 and RKE2. @slickwarren @igomez06 I'll ping you both offline once those are done for review.